packages/template-retail-react-app/app/utils/product-utils.js

```diff
@@ -200,6 +200,11 @@ export const findLowestPrice = (product) => {
         },
         {minPrice: Infinity, promotion: null, data: null}
     )
+    return {
+        ...res,
+        // when minPrice is infinity, meaning there is no variant with lowest price found, we rest the value to 0 as minPrice to be able to use this value on UI
+        minPrice: res.minPrice === Infinity ? 0 : res.minPrice
```

Comment:
How does this impact when we show the price range? Wondering if it would show `From $0` which I think is misleading

Reply:
it won't, it will only reset to 0 if the value is infinity meaning there is no variant can be found on the current product. 
![image](https://github.com/user-attachments/assets/c4f3fde3-528f-4d5b-b71f-644dc47411cc)